Tests/syntax_test_pointer.pb

```diff
@@ -113,4 +113,4 @@ Define *StringAddress = @"This is a string in memory"
 ; IDE Options = PureBasic 5.73 LTS (MacOS X - x64)
 ; CursorPosition = 20
 ; Folding = -
-; EnableXP
\ No newline at end of file
+; EnableXP
```

Comment:
I updated this test file to not contain the PureBasic IDE preferences anymore.

Reply:
In the PB IDE repository I've added a custom script that validates commits and PRs for code styles consistency, and checks PB files to ensure they don't contain settings saved in the source:

[`validate.sh`][validate.sh]


That's a Bash script, which I don't like much but at the time was my best option. I could recreate the PB settings validator as a Ruby script and add it to this repository, we could then use it for CI validation and as a Git pre-commit hook to, so checks are done locally too. 

In the PB repository I can't use Ruby because it would add an extra dependency to the project, but here we could use Ruby and Rake for all automation task.

<!----------------------------- REFERENCE LINKS ------------------------------>

[validate.sh]: https://github.com/fantaisie-software/purebasic/blob/devel/validate.sh

<!-- EOF -->